code/game/objects/items/stacks/golem_food/golem_status_effects.dm

```diff
@@ -168,7 +168,7 @@
 	alert_icon_state = "sheet-plasma"
 	alert_desc = "You are protected from high pressure and can convert heat damage into power."
 	/// What do we multiply our damage by to convert it into power?
-	var/power_multiplier = 5
+	var/power_multiplier = 5000
```

Comment:
```suggestion
	var/power_multiplier = 5 KILO WATTS
```

Reply:
It's a multiplier so we don't need units here.

Reply:
Should be joules.